src/main/java/org/javarosa/core/model/TriggerableDag.java

```diff
@@ -109,50 +109,50 @@ public TriggerableDag(EventNotifierAccessor accessor) {
         this.accessor = accessor;
     }
 
-    private Set<QuickTriggerable> doEvaluateTriggerables(FormInstance mainInstance, EvaluationContext evalContext, Set<QuickTriggerable> tv, TreeReference anchorRef, Set<QuickTriggerable> alreadyEvaluated) {
-        Set<QuickTriggerable> fired = new HashSet<>();
+    private Set<QuickTriggerable> doEvaluateTriggerables(FormInstance mainInstance, EvaluationContext evalContext, Set<QuickTriggerable> triggerables, TreeReference trigger, Set<QuickTriggerable> alreadyEvaluated) {
+        Set<QuickTriggerable> evaluated = new HashSet<>();
         // Evaluate the provided set of triggerables in the DAG's order
-        for (QuickTriggerable qt : triggerablesDAG) {
-            if (tv.contains(qt) && !alreadyEvaluated.contains(qt)) {
-                evaluateTriggerable(mainInstance, evalContext, qt, anchorRef);
-                fired.add(qt);
+        for (QuickTriggerable triggerable : triggerablesDAG) {
+            if (triggerables.contains(triggerable) && !alreadyEvaluated.contains(triggerable)) {
+                evaluateTriggerable(mainInstance, evalContext, triggerable, trigger);
+                evaluated.add(triggerable);
             }
         }
-        return fired;
+        return evaluated;
     }
 
     /**
      * Step 3 in DAG cascade. evaluate the individual triggerable expressions
      * against the anchor (the value that changed which triggered recomputation)
```

Comment:
It might be helpful to modify this comment now that there isn't an `anchor` parameter.

Reply:
All these comments need a review, honestly. I think that the new DAG documentation superseeds them, though.